src/fiatconnect/saga.ts

```diff
@@ -50,6 +68,67 @@ export function* watchFetchFiatConnectQuotes() {
   yield takeLeading(fetchFiatConnectQuotes.type, handleFetchFiatConnectQuotes)
 }
 
+export function* handleStartFiatConnectTransfer({
+  payload: params,
+}: ReturnType<typeof startFiatConnectTransfer>) {
+  const { flow, quoteId, fiatConnectQuote, fiatAccountId } = params
+
+  if (flow === CICOFlow.CashOut) {
+    try {
+      Logger.info(TAG, 'Starting transfer out..')
+      const transferResult: TransferResponse = yield call(
+        doTransferOut,
+        fiatConnectQuote,
+        fiatAccountId
+      )
+
+      Logger.info(
+        TAG,
+        'Transfer out $succeeded. Starting transaction..',
+        JSON.stringify(transferResult)
+      )
+
+      const tokenList: TokenBalance[] = yield select(tokensListSelector)
+      const tokenInfo = tokenList.find((token) => token.symbol === fiatConnectQuote.getCryptoType())
+
+      if (!tokenInfo) {
+        throw new Error(`Invalid token: ${fiatConnectQuote.getCryptoType()}`)
+      }
+
+      const feeEstimates: FeeEstimatesState['estimates'] = yield select(feeEstimatesSelector)
+      const feeInfo = feeEstimates[tokenInfo.address]?.[FeeType.SEND]?.feeInfo
+
+      const context = newTransactionContext(TAG, 'Send crypto to provider for transfer out')
+
+      const { receipt, error }: { receipt: CeloTxReceipt; error: any } = yield call(
+        buildAndSendPayment,
+        context,
+        transferResult.transferAddress,
+        new BigNumber(fiatConnectQuote.getCryptoAmount()),
+        tokenInfo.address,
+        '',
+        feeInfo!
+      )
+
+      if (error) {
+        throw error
+      }
+
+      yield put(fiatConnectTransferSuccess({ flow, quoteId, txHash: receipt.transactionHash }))
+    } catch (err) {
+      Logger.error(TAG, 'Transfer out failed..', err)
+      yield put(fiatConnectTransferFailed({ flow, quoteId }))
+    }
+  } else {
+    throw new Error('not implemented')
+  }
+}
+
+function* watchFiatConnectTransactions() {
+  yield takeEvery(startFiatConnectTransfer.type, handleStartFiatConnectTransfer)
```

Comment:
`takeLeading` stops listening after it sees `startFiatConnectTransfer.type` until `handleStartFiatConnectTransfer`. Would it maybe be good to use that for some built in rate limiting?

Reply:
Does that mean that `startFiatConnectTransfer` actions that get dispatched while one is already running would get silently ignored? No risk of that breaking anything now (or probably ever), just wondering.

Reply:
Yep, silently ignores other requests while handleStartFiatConnectTransfer is running